code/modules/food&drinks/food/snacks/meat.dm

```diff
@@ -200,6 +209,10 @@
 /obj/item/weapon/reagent_containers/food/snacks/meat/steak/spider
 	name = "spider steak"
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/steak/goliath
+	name = "goliath steak"
+	desc = "A delicious, lava cooked steak."
+	burn_state = LAVA_PROOF
```

Comment:
I don't know of any interesting reagents honestly. What are some good ones?

Keep in mind goliath corpses are very common and drop armour anyway, this is more a fluff/fun thing than anything else.